separate_negative_positive_nos.py

```diff
@@ -0,0 +1,27 @@
+#to separate the positive and negative numbers of the list
+
+def Pos_Neg(L,n):
```

Comment:
Add docstring definition of functions.
Follow PEP8 Guidelines for function and variable naming
https://www.python.org/dev/peps/pep-0008/